packages/@okta/vuepress-site/docs/guides/submit-app/main/wfconnector/submit.md

```diff
@@ -0,0 +1,23 @@
+Submit a specific Workflows connector version from your Workflows Connector Builder environment in your org.
+
+#### Connector details
+
+* **Connector**: Select the Workflows connector you want to submit.
+
+* **Version**: Select the version of Workflows connector you want to submit.
+
+The list of available connectors and the associated versions are based on the Workflows Connector Builder environment from the org associated with your user credential. These fields are automatically populated if you submit from the Workflows Connector Builder.
+
+#### Validation
+
+A flow test suite is required to validate the functionality of your connector. This test suite is obtained from exporting your flows in the Workflows environment. See [Export Folder](https://help.okta.com/wf/en-us/Content/Topics/Workflows/function-reference/Folders/folders_exportfolder.htm).
+
+* Click **Download file** to download the flows test suite in this submission. This file is automatically populated from your Workflows Connector environment if you have flows associated with your submitted connector.
```

Comment:
same as above, descoped for GA

Reply:
Also its untested whether CB can actually upload file to ISV Portal at the time of writing as it's not developed/tested.

Reply:
ok, thanks for letting me know.